modules/wrapper/lazy/lazy.nix

```diff
@@ -36,7 +36,6 @@
       mode = mkOption {
         description = "Modes to bind in";
         type = either str (listOf str);
-        default = ["n" "x" "s" "o"];
       };
```

Comment:
Maybe make this an example instead.

Also, please move description below type and default.